packages/server/graphql/mutations/downgradeToStarter.ts

```diff
@@ -42,6 +60,26 @@ export default {
     }
 
     // RESOLUTION
+    if (!isSuperUser) {
+      const [billingLeaders, organization] = await Promise.all([
+        getBillingLeadersByOrgId(orgId, dataLoader),
+        dataLoader.get('organizations').load(orgId)
+      ])
+      if (!billingLeaders[0]) {
+        const error = new Error('Unable to find billing leader')
+        sendToSentry(error, {userId: viewerId})
+        return
+      }
+      analytics.organizationDowngraded(viewerId, {
+        orgId,
+        oldTier: tier,
+        newTier: 'starter',
+        billingLeaderEmail: billingLeaders[0].email,
+        orgName: organization.name,
+        reasonsForLeaving,
+        otherTool
```

Comment:
-1 Maybe set some limit on that field? If I put to big value, the code fails with an error

```
22:04:35 1|GraphQL Executor  | You have triggered an unhandledRejection, you may have forgotten to catch a Promise rejection:
22:04:35 1|GraphQL Executor  | AssertionError [ERR_ASSERTION]: Your message must be < 32kb.
```

Reply:
I like it, updated 👍